specification/k3po.control/SPEC.md

```diff
@@ -29,60 +47,20 @@ If the "content-length" header is specified, then a message body of the specifie
-ERROR event is sent with the cause of the failure.
-
-- If there is no PREPARE command sent before the START command or if the PREPARE command was not successful an error with a message "Script has
- not been prepared or is still preparing" must occur. 
-
-- If the START command was already sent, an error with a message "Script has already been started" must occur.
-
-- Any time after the PREPARE command and before the DISPOSE command, the test framework can send an **AWAIT** command for a barrier existing 
-in the scripts. The command receives a response with a NOTIFIED event only when the corresponding barrier is notified. The NOTIFIED event may
- never be returned in case the barrier is never triggered. In case the barrier does not exist, an error must be returned.
-
-- Any time after the PREPARE command and before DISPOSE, the test framework can send a **NOTIFY** command for a barrier existing in the 
-scripts. The command notifies the barrier and a response with a NOTIFIED event is received. In case the barrier does not exist, an error must 
-be returned.
-
-- Any time after the PREPARE command and before DISPOSE, the test framework can send an **ABORT** command. The driver must abort the current 
-execution. If the execution of the scripts is already finished, no event is sent back. If the execution is aborted by this command, a FINISHED 
-event with the script that was executed so far must be returned.
-
-- Any time after the PREPARE command, the test framework can send a **DISPOSE** command. The driver must abort the execution and release the 
-resources in use. When done, a DISPOSED event must be sent back. At this point you can consider that the test has ended and the driver is back in the initial state.
-
-- After a DISPOSE command, no other command is accepted except a new PREPARE. Any other command will have as a result an error with a message "Script has not been prepared or is still preparing". 
-
+## Message Types
 
 ### Commands
 
 ##### PREPARE command
 
 ###### Description
-This command will initiate a new conversation. Scripts that should be loaded are given in the **name** headers. Additional override values for
-properties existing in the scripts can be given in the message body.
+This **PREPARE** command informs the driver to prepare one or more K3PO script for execution. 
 
 ###### Headers
 - version - the version of the protocol. Currently the only supported version is 2.0.
 - content-length - the length of the message body. This header is optional, should be present only when message body is not empty.
 - origin - the root folder for the scripts to be executed. This header is optional
-- name - the name of the script to be executed. It is a relative path to the origin. Can appear more than once (several scripts to be executed).
+- name - the name of the scripts to be executed. It is a relative path to the origin. Can appear more than once (several scripts to be executed).
 
```

Comment:
In one 'name' header, there is the name of one script and only one. If you have multiple scripts, the 'name' header will occur more than once, for each of the scripts. So I think that initial version is correct.